css/properties/position-try-options.json

```diff
@@ -0,0 +1,40 @@
+{
+  "css": {
+    "properties": {
+      "position-try-options": {
```

Comment:
Missing sub features: flip-block, flip-inline, flip-start, none.

Reply:
I've added the list value subfeatures. However, this property can also take an `inset-area` value as a value, to use as a position try option, and it can take a dashed ident as an option, which represents a custom position try option defined inside a `@position-try` at-rule. Do we not need separate sub-features for these value options?

Reply:
I think it would be great to add them in but I'm not going to block this PR if they aren't here for now.